javascript/drizzle-orm/security/audit/ban-drizzle-sql-raw.yaml

```diff
@@ -0,0 +1,91 @@
+rules:
+  - id: ban-drizzle-sql-raw
+    message:
+      Use of sql.raw() with SQL string concatenation is not allowed as it may lead to SQL injection.
+      Use parameterized queries with Drizzle's query builders instead.
+    fix: "/* Use sql`...` template literals or parameterized queries instead */"
+    metadata:
+      category: security
+      references:
+        - https://orm.drizzle.team/docs/sql#sqlraw
+      technology:
+        - javascript
+        - typescript
+        - drizzle-orm
+      cwe:
+        - "CWE-89: Improper Neutralization of Special Elements used in an SQL Command ('SQL Injection')"
+      subcategory:
+        - audit
+      likelihood: LOW
+      impact: HIGH
+      confidence: LOW
+    languages:
+      - javascript
+      - typescript
+    severity: ERROR
+    pattern-either:
+      - patterns:
+          - pattern-either:
+              - pattern-inside: |
+                  const { sql } = require('drizzle-orm')
```

Comment:
Explicit `const/var/let` here necessitates coverage of the other two, unless you are certain that this will only ever be a `const` import. Not making this change will cause CI to throw linter errors on this rule.